core/src/com/unciv/logic/GameStarter.kt

```diff
@@ -133,7 +133,7 @@ object GameStarter {
         val startNanos = System.nanoTime()
         action()
         val delta = System.nanoTime() - startNanos
-        println("GameStarter.$text took ${delta/1000000L}.${(delta/10000L).rem(100)}ms")
+        debug("GameStarter.%s took %s.%sms", text, delta/1000000L, (delta/10000L).rem(100))
```

Comment:
This goes to an old-school C-style print? Won't that balk at longs passed for %s?